modules/core/pom.xml

```diff
@@ -453,7 +453,7 @@
                                         <artifactItem>
                                             <groupId>javax.cache</groupId>
                                             <artifactId>app-domain</artifactId>
-                                            <version>${javax.cache.tck.version}</version>
+                                            <version>${javax.cache.version}</version>
```

Comment:
A simple bug in current pom file, which is hidden in master. You can restore this change and will see the problem.